response_operations_ui/controllers/case_controller.py

```diff
@@ -103,8 +103,11 @@ def get_cases_by_business_party_id(business_party_id):
 
     try:
         response.raise_for_status()
+        if response.status_code == 204:
+            logger.info('No cases found for business', business_party_id=business_party_id)
+            return []
```

Comment:
Is this (or any of the changes) going to work?  On success it returns response.json (a dictionary), on a 404 or 204, it returns a list.